storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -1023,6 +1025,14 @@ struct my_option xb_client_options[] =
     "privileges before performing any query.", &opt_check_privileges,
    &opt_check_privileges, 0, GET_BOOL, NO_ARG, 0, 0, 0, 0, 0, 0},
 
+  {"read_buffer_size",
+   OPT_XTRA_DATAFILE_READ_BUFFER_MULTIPLICATOR,
+   "Set datafile read buffer size, given value is scaled up to page size."
+   " Use 0 for default, which is 10Mb.",
```

Comment:
One more thing - constant `OPT_XTRA_DATAFILE_READ_BUFFER_MULTIPLICATOR ` should be changed to match option name.
OK. Lets compare. User read this description as "set 0 if you want 10Mb". When user run xtrabackup without arguments, xtrabackup will output default values (and ones from my.cnf) and user will see "read_buffer_size=0". It makes confusion.
Dependency. When you change default value you are going to change help text as well. Isn't it the same level of dependency as change the value itself? Actually I think it is less obvious dependency and one which much easier to miss.
Is there any specific cases when we want to distinguish user-set value and default one? If there are, then we *must* do that in `get_one_option` and not rely on magic value of 0.